Emby.Drawing/ImageProcessor.cs

```diff
@@ -137,14 +137,14 @@ public async Task ProcessImage(ImageProcessingOptions options, Stream toStream)
             }
         }
 
-        public ImageFormat[] GetSupportedImageOutputFormats()
-        {
-            return _imageEncoder.SupportedOutputFormats;
-        }
+        public IReadOnlyCollection<ImageFormat> GetSupportedImageOutputFormats()
+            => _imageEncoder.SupportedOutputFormats;
+
+        private static readonly HashSet<string> TransparentImageTypes
```

Comment:
Why this isn't declared as IRradOnlyCollection?

Reply:
Cause it’s private, so it’ll never be another type.